### ISSUE TITLE
send more files into the analysis server as overlays

### ISSUE THREAD

**Issue by devoncarew** [dart-atom/dartlang#437]
I.e., not just `.dart` files. We'll need a way to figure out what files. Currently it would look something like `.analysis_options`, `*.dart`, `*.html`.

**Comment by devoncarew** [comment]
@pq, @scheglov, @bwilkerson

**Comment by pq** [comment]
Yes please!  It would be great to see incremental analysis of `.analysis_options`.

Thanks!

**Comment by devoncarew** [comment]
We're sending in `*.dart` and `.analysis_options`; we should discuss a way to get a set of path globs for interesting files from the analysis server :)

**Comment by pq** [comment]
Cool!

I opened an issue https://github.com/dart-lang/sdk/issues/24746 to track this idea.
